mempool/mempool.go

```diff
@@ -927,8 +927,7 @@ func (mp *TxPool) validateReplacement(tx *btcutil.Tx,
 func (mp *TxPool) maybeAcceptTransaction(tx *btcutil.Tx, isNew, rateLimit, rejectDupOrphans bool) ([]*chainhash.Hash, *TxDesc, error) {
 	txHash := tx.Hash()
 
-	// If a transaction has witness data, and segwit isn't active yet, If
-	// segwit isn't active yet, then we won't accept it into the mempool as
+	// If a transaction has witness data, and segwit isn't active yet, then we won't accept it into the mempool as
```

Comment:
Please break after 80 characters (while using 8 spaces for any tab character)

Reply:
Done.